targets/frontend/src/pages/alerts/[[...params]].js

```diff
@@ -93,13 +93,28 @@ export function AlertPage() {
 
   function getTitle(alert) {
     if (alert.info.num) {
-      return `IDCC ${alert.info.num} - ${new Date(
-        alert.created_at
-      ).toLocaleDateString()} (${alert.ref})`;
+      return (
+        <>
+          IDCC {alert.info.num}
+          <Text
+            title={alert.info.title}
+            sx={{
+              fontVariantPosition: "super",
+              marginLeft: 4,
```

Comment:
I'm not sure about this one. Theme-UI seems to use the value from the theme but I didn't saw margin values.

Reply:
Why not use an Icon here (with a aria-label) we use react-icons on the project

Reply:
also margin are expressed in small / medium / large

Reply:
I've changed to react-icons:
<img width="723" alt="CleanShot 2021-02-26 at 14 09 39@2x" src="https://user-images.githubusercontent.com/992514/109306224-28efa980-783f-11eb-82c9-e2895654ca60.png">